autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillSettingsViewModel.kt

```diff
@@ -62,6 +67,9 @@ class AutofillSettingsViewModel @Inject constructor(
     private val _commands = MutableStateFlow<List<Command>>(emptyList())
     val commands: StateFlow<List<Command>> = _commands
 
+    private val _commandsCredentialView = MutableStateFlow<List<CredentialModeCommand>>(emptyList())
```

Comment:
I found it necessary to differentiate between the existing `commands` that the `AutofillManagementActivity` might consume vs commands that were expected to be consumed by the fragment only. 

Re-using the same commands is error prone, as both activity and fragment could try to consume it but only one would get it. Adding specific commands that only the credential view fragment would consume solves this. (I don't _love_ the approach, but it works)